spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -21,7 +21,6 @@ bom {
 					"activemq-blueprint",
 					"activemq-broker",
 					"activemq-client",
-					"activemq-client-jakarta",
```

Comment:
This should remain so that users' builds aren't broken if they have a dependency on `activemq-client-jakarta`.

Reply:
Fixed